src/libs/hooks/__tests__/useShowReview.test.tsx

```diff
@@ -98,4 +114,32 @@ describe('useShowReview', () => {
       expect(mockUpdateInformationWhenReviewHasBeenRequested).not.toHaveBeenCalled()
     })
   })
+
+  describe('FF disableStoreReview', () => {
+    it('should not show the review when we disabled store review', () => {
+      mockIsAvailable.mockReturnValueOnce(true)
+      mockUseReviewInAppInformation.mockReturnValueOnce({ shouldReviewBeRequested: true })
+      mockRequestInAppReview.mockResolvedValueOnce(undefined)
+      mockSettings.disableStoreReview = false
+
+      render(<TestReviewComponent />)
+
+      jest.advanceTimersByTime(3000)
+
+      expect(mockRequestInAppReview).toHaveBeenCalledWith()
+    })
+
+    it('should not show review when we enabled store review', () => {
+      mockIsAvailable.mockReturnValueOnce(true)
+      mockUseReviewInAppInformation.mockReturnValueOnce({ shouldReviewBeRequested: true })
+      mockRequestInAppReview.mockResolvedValueOnce(undefined)
+      mockSettings.disableStoreReview = true
+
+      render(<TestReviewComponent />)
+
+      jest.advanceTimersByTime(3000)
+
+      expect(mockRequestInAppReview).not.toHaveBeenCalledWith()
```

Comment:
ce qu'on cherche à vérifier, c'est que ça n'a pas été appelé

```suggestion
      expect(mockRequestInAppReview).not.toHaveBeenCalled()
```